library/tiqr/OATH/OCRAParser.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once ( __DIR__ . "/../Tiqr/Random.php");
```

Comment:
~Was injecting the `Random` helper not an option? That way we can keep coupling low.~

I later learned this helper class distributes a set of static helper functions. Defeating my previous DI suggestion. But leaving the increased coupling issue at hand.

Reply:
I agree that using DI is the proper technique. My overall approach is to improve the library in small steps in areas where it counts for how the library is used in practice while adding tests in the process. I chose to keep following the style of the library and tackle DI at another time. My reasoning was that this at least allows Tiqr_Random class to be mocked in tests.